app/javascript/flavours/polyam/features/notifications_v2/components/notification_follow.tsx

```diff
@@ -14,18 +18,49 @@ const labelRenderer: LabelRenderer = (values) => (
   />
 );
 
+const FollowerCount: React.FC<{ accountId: string }> = ({ accountId }) => {
+  const account = useAppSelector((s) => s.accounts.get(accountId));
+
+  if (!account) return null;
+
+  return (
+    <ShortNumber value={account.followers_count} renderer={FollowersCounter} />
+  );
+};
+
 export const NotificationFollow: React.FC<{
   notification: NotificationGroupFollow;
   unread: boolean;
-}> = ({ notification, unread }) => (
-  <NotificationGroupWithStatus
-    type='follow'
-    icon={PersonAddIcon}
-    iconId='person-add'
-    accountIds={notification.sampleAccountIds}
-    timestamp={notification.latest_page_notification_at}
-    count={notification.notifications_count}
-    labelRenderer={labelRenderer}
-    unread={unread}
-  />
-);
+}> = ({ notification, unread }) => {
+  let actions: JSX.Element | undefined;
+  let additionalContent: JSX.Element | undefined;
+
+  if (notification.sampleAccountIds.length === 1) {
+    // only display those if the group contains 1 account, otherwise it does not make sense
+    const account = notification.sampleAccountIds[0];
+
+    if (account) {
+      actions = (
+        <FollowIconButton accountId={notification.sampleAccountIds[0]} />
+      );
+      // Polyam: Don't show follower count
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      additionalContent = undefined && <FollowerCount accountId={account} />;
```

Comment:
Instead of just hiding the follow count, this could be changed to show actual useful information.
Upstream is somewhat reluctant to do that because of hover cards, but when those are disabled, there is basically no info.